### PR TITLE
BUX-589/migrate package

### PR DESCRIPTION
**Pull request by Nazarii-4chain**
Changed names
# Pull Request Checklist

- [ ] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/BuxOrg/js-buxclient/blob/main/.github/CODE_STANDARDS.md)
- [ ] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/BuxOrg/js-buxclient/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [ ] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,17 +124,16 @@ update-contributors           Regenerates the contributors html/list
...
-Here's the [getting started](https://getbux.io) with BUX
 
 ```javascript
 import bsv from 'bsv';
-import { BuxClient } from 'js-buxclient';
+import { SpvWalletClient } from 'spv-wallet-js-client';
 
 const server = "http://localhost:3003/v1";
 const transportType = 'http'; // or graphql
 const xPriv = bsv.HDPrivateKey.fromRandom();
```
Comment (dorzepowski) [inline README.md:134]: we should suggest generating keys from this lib, not from bsv lib.

File: README.md [hunk 2]
```diff
@@ -62,13 +48,13 @@ View more [BUX documentation](https://getbux.io).
...
-### Connection as a BUX admin
+### Connection as an SPV Wallet admin
 
-You can add the admin key to the BUX client after initialisation to support admin requests (IE: registering an xPub).
+You can add the admin key to the SPV Wallet client after initialisation to support admin requests (IE: registering an xPub).
 
 ```javascript
 const adminKey = 'xprv.....';
-buxClient.SetAdminKey(adminKey);
+spvWalletClient.SetAdminKey(adminKey);
```
Comment (dorzepowski) [inline README.md:57]: IMHO as a variable name, we can use just a `client` in most of the code.
As mostly the name of variable is in some context, for example in context of creation, like
```
const client = new SpvWalletClient(...)
```

**Review by dorzepowski — CHANGES_REQUESTED** [review]

File: .github/workflows/test(ubuntu-latest).yml
[the remark this answers is not in the document]
Comment (Nazarii-4chain) [reply .github/workflows/test(ubuntu-latest).yml:1]: I copied this from another workflow. It's clear that tests runs only on ubuntu. Anyway I can rename this.

File: README.md (hunk 1 above)
Comment (Nazarii-4chain) [reply README.md:134]: Changed

File: README.md (hunk 2 above)
Comment (Nazarii-4chain) [reply README.md:57]: Changed

File: LICENSE [hunk 3]
```diff
@@ -1,5 +1,5 @@
 Open BSV License
-Copyright (c) 2022 BuxOrg
+Copyright (c) 2022 SPV Wallet
```
[the remark this answers is not in the document]
Comment (Nazarii-4chain) [reply LICENSE:2]: Changed

File: .github/workflows/test(ubuntu-latest).yml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        with:
+          cmd: install # will run `yarn install` command
+      - name: Build production bundle
+        uses: borales/actions-yarn@v4
+        with:
+          cmd: build # will run `yarn build` command
+      - name: Run tests
+        uses: borales/actions-yarn@v4
+        with:
+          cmd: test # will run `yarn test` command
```
[the remark this answers is not in the document]
Comment (Nazarii-4chain) [reply .github/workflows/test(ubuntu-latest).yml:35]: Thanks. Changed

File: .github/workflows/test(ubuntu-latest).yml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+          registry-url: https://registry.npmjs.org/
```
[the remark this answers is not in the document]
Comment (Nazarii-4chain) [reply .github/workflows/test(ubuntu-latest).yml:23]: Added

File: .github/workflows/test(ubuntu-latest).yml [hunk 6]
```diff
@@ -0,0 +1,35 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: test(ubuntu-latest)
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
```
[the remark this answers is not in the document]
Comment (Nazarii-4chain) [reply .github/workflows/test(ubuntu-latest).yml:10]: Removed

File: .github/CODE_STANDARDS.md [hunk 7]
```diff
@@ -352,9 +352,8 @@ Additional information and guidelines on Conventional Commits can be found [here
 
 Good example:
 
-```bash
-feat: add possibility to create a new user by admin (#BUX-123)
-```
+feat(#123): add possibility to create a new user by admin
```
[the remark this answers is not in the document]
Comment (Nazarii-4chain) [reply .github/CODE_STANDARDS.md:355]: Added

File: .all-contributorsrc
[the remark this answers is not in the document]
Comment (Nazarii-4chain) [reply .all-contributorsrc:1]: Removed

File: .github/workflows/test(ubuntu-latest).yml
[the remark this answers is not in the document]
Comment (dorzepowski) [reply .github/workflows/test(ubuntu-latest).yml:1]: I don't think that anyone cares on what machine it is run in most cases. And short, concise, clear name is better.

[the remark this answers is not in the document]
Comment (Nazarii-4chain) [reply .github/workflows/test(ubuntu-latest).yml:1]: Renamed

File: .github/workflows/test(ubuntu-latest).yml [hunk 8]
```diff
@@ -0,0 +1,35 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: test(ubuntu-latest)
```
[the remark this answers is not in the document]
Comment (Nazarii-4chain) [reply .github/workflows/test(ubuntu-latest).yml]: Renamed

File: README.md [hunk 9]
```diff
@@ -138,17 +124,15 @@ update-contributors           Regenerates the contributors html/list
...
 
 ```javascript
-import bsv from 'bsv';
-import { BuxClient } from 'js-buxclient';
+import { SpvWalletClient } from 'spv-wallet-js-client';
 
 const server = "http://localhost:3003/v1";
 const transportType = 'http'; // or graphql
-const xPriv = bsv.HDPrivateKey.fromRandom();
+const xPriv = generateKeys().xPriv()
```
[the remark this answers is not in the document]
Comment (Nazarii-4chain) [reply README.md:133]: Added import

**Review by dorzepowski — APPROVED** [review]

**Comment by kuba-4chain** [comment]
Testing...

**Pull request merged**